txircd/modules/extra/denychans.py

```diff
@@ -13,6 +15,17 @@ class DenyChannels(ModuleData):
 	
 	def actions(self):
 		return [ ("joinpermission", 50, self.blockNonDenied) ]
+
+	def verifyConfig(self, config):
+		for option in ("deny_channels", "allow_channels"):
+			if option in config:
+				if not isinstance(config[option], list):
+					raise ConfigValidationError(option, "value must be a list")
+				for chanName in config[option]:
+					if chanName[0] != "#":
+						chanName = "#{}".format(chanName)
+					if not isValidChannelName(chanName):
```

Comment:
This breaks wildcards for channel names. It's probably easier not to check this.